docs/book/README.md

```diff
@@ -0,0 +1,26 @@
+# Kubernetes vSphere CSI Driver
+
+The [Container Storage Interface (CSI)](https://github.com/container-storage-interface/spec/blob/master/spec.md) is a specification designed to enable persistent storage volume management on Container Orchestrators (COs) such as Kubernetes. This allows storage systems to integrate with containerized workloads running on Kubernetes. Using CSI, third-party storage providers (like VMware) can write and deploy plugins for storage systems in Kubernetes without a need to modify any core Kubernetes code.
```

Comment:
I think the home page needs to have more vSphere specific information for users. Most users will probably already kno what CSI is but will be curious how this one is different and what vSphere capabilities it's leveraging.

Reply:
yes agree. let me then just keep an index here to allow navigating mark down files, if some one is not using generated html site.

We will be adding vSphere CSI driver related overview and introduction at `docs/book/overview.md`.